app/controllers/api/custom_button_sets_controller.rb

```diff
@@ -7,5 +7,12 @@ def create_resource(type, id, data)
     def edit_resource(type, id, data)
       super(type, id, data.deep_symbolize_keys)
     end
+
+    def order_resource(_, _, data)
```

Comment:
@akhilkr128 As mentioned elsewhere, I'm thinking this should be `reorder`.  I'm concerned the term `order` might get confused with things like ordering services.